tuna/worker_interface.py

```diff
@@ -200,56 +218,56 @@ def check_jobs_found(self, job_rows, find_state, imply_end):
       return False
     return True
 
-  def get_job_from_tuple(self, job_tuple):
+  def get_job_from_tuple(self, job_tuple: SimpleDict) -> Optional[SimpleDict]:
```

Comment:
incorrect typing
purpose of this function is to extract the SimpleDict - job object from job_tuple
job_tuple in this case is assumed to either be Tuple[SimpleDict, ...] or SimpleDict